server/endpoints/Review.test.ts

```diff
@@ -0,0 +1,85 @@
+import mongoose from 'mongoose';
+import { MongoMemoryServer } from 'mongodb-memory-server';
+import express from "express";
+
+import axios from 'axios';
+import { configure } from "../endpoints";
+import { Classes, Reviews } from "../dbDefs";
+
+let mongoServer: MongoMemoryServer;
+let serverCloseHandle;
+
+const testingPort = 8000;
+
+beforeAll(async () => {
+  // get mongoose all set up
+  mongoServer = new MongoMemoryServer();
+  const mongoUri = await mongoServer.getUri();
+  await mongoose.connect(mongoUri, { useNewUrlParser: true, useUnifiedTopology: true });
+
+  const c2110 = new Classes({
+    _id: "oH37S3mJ4eAsktypy",
+    classSub: "cs",
+    classNum: "2110",
+    classTitle: "Object-Oriented Programming and Data Structures",
+    classPrereq: [],
+    classFull: "cs 2110 object-oriented programming and data structures",
+    classSems: ["FA14", "SP15", "SU15", "FA15", "SP16", "SU16", "FA16", "SP17",
+      "SU17", "FA17", "SP18", "FA18", "SU18", "SP19", "FA19", "SU19"],
+    crossList: ["q75SxmqkTFEfaJwZ3"],
+    classProfessors: ["David Gries", "Douglas James", "Siddhartha Chaudhuri",
+      "Graeme Bailey", "John Foster", "Ross Tate", "Michael George",
+      "Eleanor Birrell", "Adrian Sampson", "Natacha Crooks", "Anne Bracy",
+      "Michael Clarkson"],
+    classDifficulty: 2.9,
+    classRating: null,
+    classWorkload: 3,
+  });
+
+  const review1 = new Reviews({
+    _id: "4Y8k7DnX3PLNdwRPr",
+    text: "review text for cs 2110",
+    difficulty: 1,
+    quality: 4,
+    class: "oH37S3mJ4eAsktypy",
+    grade: 6,
+    date: new Date().toISOString(),
+    atten: 0,
+    visible: 1,
+    reported: 0,
+  });
+
+  const review2 = new Reviews({
+    _id: "4Y8k7DnX3PLNdwRPq",
+    text: "review text for cs 2110 number 2",
+    difficulty: 1,
+    quality: 5,
+    class: "oH37S3mJ4eAsktypy",
+    grade: 6,
+    date: new Date().toISOString(),
+    atten: 0,
+    visible: 1,
+    reported: 0,
+  });
+
+  review2.save();
```

Comment:
This returns a Promise. Although it doesn't seem to cause any issues, it should probably still be awaited.

Reply:
Same for the next two lines

Reply:
This is fixed, resolving